test/unit/messaging/message-schema.test.js

```diff
@@ -0,0 +1,34 @@
+const schema = require('../../../app/messaging/message-schema')
+const { REQUEST_MESSAGE } = require('../../mocks/messaging/message')
+
+let message
+
+describe('message schema', () => {
+  beforeEach(() => {
+    message = REQUEST_MESSAGE
+  })
+
+  test('should validate a valid message', () => {
+    expect(schema.validate(message).error).toBeUndefined()
+  })
+
+  test('should not validate a message with an invalid body', () => {
+    message.body = 'invalid'
+    expect(schema.validate(message).error).toBeDefined()
+  })
+
+  test('should not validate a message with a missing category', () => {
+    delete message.body.category
+    expect(schema.validate(message).error).toBeDefined()
+  })
+
+  test('should not validate a message with an invalid category', () => {
+    message.body.category = 'invalid'
+    expect(schema.validate(message).error).toBeDefined()
+  })
```

Comment:
Swap these two tests around to keep the invalid's together and the missing's together?